src/linked_list.py

```diff
@@ -0,0 +1,91 @@
+# coding=utf-8
+from __future__ import unicode_literals, print_function
+from builtins import *
```

Comment:
Never ever ever ever ever import *
Also... This doesn't even work. It broke the build, and when I removed it, everything worked. Why is it here?